addons/account/models/account_move.py

```diff
@@ -3385,6 +3394,10 @@ def _copy_data_extend_business_fields(self, values):
         '''
         self.ensure_one()
 
+        # Don't copy the name of a payment term line.
+        if self.move_id.is_invoice() and self.account_id.user_type_id.type in ('receivable', 'payable'):
+            values['name'] = ''
+
```

Comment:
Hm, it says include_business_fields and we don't copy them.  Should not it go in the method below?